### PR TITLE
Implemented onBufferEnd cb prop for FilePlayer, YouTube and Facebook

### PR DESCRIPTION
**Pull request by me-andre**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,6 +92,7 @@ Prop | Description
 `onDuration` | Callback containing duration of the media, in seconds
 `onPause` | Called when media is paused
 `onBuffer` | Called when media starts buffering
+`onBufferEnd` | Called when media has finished buffering (works for file paths, YouTube and Facebook players)
```
Comment (CookPete) [inline README.md]:
```suggestion
`onBufferEnd` | Called when media has finished buffering<br />&nbsp; ◦ &nbsp;Works for files, YouTube and Facebook
```

**Comment by CookPete** [comment]
One suggested tweak to the readme, and `onBufferEnd` needs to be added to `index.d.ts`, but otherwise I think we are good 👍

**Comment by me-andre** [comment]
done :)

**Comment by CookPete** [comment]
Nice work 👍 Apologies for the delay.

**Pull request merged**